### PR TITLE
Remove unnecessary validator code

### PR DESCRIPTION
**Pull request by ifrit98**
This PR removes validator related code as it now lives in `openvalidators` repo.

### CODE REVIEW

File: bittensor/_cli/commands/misc.py [hunk 1]
```diff
@@ -28,13 +28,6 @@ class HelpCommand:
     @staticmethod
     def run (cli):
         sys.argv = [sys.argv[0], '--help']
-        # # Run miner.
-        # if cli.config.model == 'core_server':
```
Comment (camfairchild) [inline bittensor/_cli/commands/misc.py:32]: Should remove btcli run then?

**Review by camfairchild — COMMENTED** [review]
See comment

File: bittensor/_cli/commands/misc.py (hunk 1 above)
Comment (ifrit98) [reply bittensor/_cli/commands/misc.py:32]: Might as well gut the whole HelpCommand class, ya?

Comment (camfairchild) [reply bittensor/_cli/commands/misc.py:32]: > Might as well gut the whole HelpCommand class, ya?

Yes for sure

**Review by camfairchild — APPROVED** [review]

**Pull request merged**